### PR TITLE
Porting Adaptive Templates to Xunit

### LINKED ISSUE #4092: Migration builder.dialogs.adaptive.templates
#4039

### PR DESCRIPTION
**Pull request by Virtual-Josh**
Fixes #4092

### CODE REVIEW

File: tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs [hunk 1]
```diff
@@ -451,6 +457,14 @@ public async Task TestNoResourceExplorerLanguageGeneration()
             .StartTestAsync();
         }
 
+        internal static void Initialize()
+        {
+            ComponentRegistration.Add(new DeclarativeComponentRegistration());
+            ComponentRegistration.Add(new AdaptiveComponentRegistration());
+            ComponentRegistration.Add(new AdaptiveTestingComponentRegistration());
+            ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
+        }
```
Comment (tomlm) [inline tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs:466]: This only should be run once, not with each unit test.  Old test was [AssemblyInitialize]

File: tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Startup.cs [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-    public class Startup
-    {
-        [AssemblyInitialize]
-        public static void Initialize(TestContext testContext)
-        {
-            ComponentRegistration.Add(new DeclarativeComponentRegistration());
-            ComponentRegistration.Add(new AdaptiveComponentRegistration());
-            ComponentRegistration.Add(new AdaptiveTestingComponentRegistration());
-            ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
-        }
```
Comment (tomlm) [inline tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Startup.cs:19]: This only runs once for ALL classes in the assembly.

File: tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs (hunk 1 above)
Comment (Virtual-Josh) [reply tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/LGGeneratorTests.cs]: updated

File: tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Startup.cs (hunk 2 above)
Comment (Virtual-Josh) [reply tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Startup.cs:19]: updated

**Pull request merged**